datahost-ld-openapi/test/tpximpact/datahost/ldapi/datagen/reports.clj

```diff
@@ -0,0 +1,107 @@
+(ns tpximpact.datahost.ldapi.datagen.reports
+  ;;Test dataset generation for load/performace experiments and testing. 
+  ;; Example usage-  Creating a dataset of 100,000 rows and 15 columns:
+  ;; (gen/get-data 15 100000 "test/tpximpact/datahost/ldapi/datagen/hurl_tests/" "FILENAME")
+  (:require [tpximpact.datahost.ldapi.datagen.load-test :as load-test]))
+
+
+(def seed (System/currentTimeMillis))
+;; Documentation
+
+;; GENERATE DATA
+;; - (load-test/generate x y repeats filename seed)
+;; x: int number of desired columns
+;; y: int number of desired rows
+;; repeats: int (only relevant for AD)
+;; filename: string with no spaces
+;; seed: int used for regenerating the same dataset again
+
+;; This will generate a schema and corresponding csv in datagen/issue-310
+;; It will also generate two hurl scripts:
+;; A: Append, tests a single append
+;; AD: Append Delete, appends and deletes the csv for as many repeats as specified
+
+;; GENERATE SPLIT DATA
+;; - (load-test/generate-split x y splits filename seed repeats)
+;; splits: int how many csv's the data should be divided into
+
+;; This will generate a schema and a number corresponding csv's in datagen/issue-310
+;; It will also generate a hurl script:
+;; CA: Consecutive Appends, posts all the split csvs one after the other
+
+;; NOTES
+;; - The command for running the hurl scripts can be found at the top of each one
+;; - It takes a few minutes to create data for 10mil, let it run or the csv will be incomplete.
+
+;; report on some basic cases -
+
+(load-test/generate 15 10000000 1 "tenmil" seed)
```

Comment:
wrap those `load-test/generate` calls in `(comment ...)` or function. We don't want to accidentally generate hundreds MB of data just just by `require`ing this namespace.